packages/astro/actions-module.template.mjs

```diff
@@ -0,0 +1,49 @@
+import { ActionError, callSafely } from 'astro:actions';
+
+function toActionProxy(actionCallback = {}, aggregatedPath = '/_actions/') {
```

Comment:
Why is this file here and not inside `src/`? Also, I think it's a top-level comment that explains this file

Reply:
@florian-lefebvre That's right. I noticed you created a `templates/` directory for these files in your env PR, so I'll preemptively move there